core/tests/protractor_utils/TopicsAndSkillsDashboardPage.js

```diff
@@ -245,32 +245,31 @@ var TopicsAndSkillsDashboardPage = function() {
   this.filterSkillsByStatus = async function(status) {
     await action.click(
       'Skill Dashboard status filter', skillStatusFilterDropdown);
-    await (
-      await browser.driver.switchTo().activeElement()
-    ).sendKeys(status + '\n');
+    var dropdownOption = element(
+      by.cssContainingText('mat-option .mat-option-text', status));
```

Comment:
Is there a parent element with class `protractor-test-*` that we can reference? Then you could chain selectors as described on the [e2e wiki page](https://github.com/oppia/oppia/wiki/End-to-End-Tests#selecting-elements)

Reply:
done

Reply:
No, this clickable element has no parent with class protractor-test-*. This element is direct descendent of body tag. So, can't chain it